Demo/src/main/java/com/braintreepayments/demo/PaymentButtonsFragment.java

```diff
@@ -1,26 +1,48 @@
 package com.braintreepayments.demo;
 
+import android.net.Uri;
 import android.os.Bundle;
+import android.text.TextUtils;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.fragment.app.FragmentActivity;
+import androidx.navigation.NavDirections;
+import androidx.navigation.fragment.NavHostFragment;
 
+import com.braintreepayments.api.core.UserCanceledException;
 import com.braintreepayments.api.uicomponents.PayPalButton;
 import com.braintreepayments.api.uicomponents.PayPalButtonColor;
 import com.braintreepayments.api.uicomponents.VenmoButton;
 import com.braintreepayments.api.uicomponents.VenmoButtonColor;
+import com.braintreepayments.api.venmo.VenmoAccountNonce;
+import com.braintreepayments.api.venmo.VenmoClient;
+import com.braintreepayments.api.venmo.VenmoLauncher;
+import com.braintreepayments.api.venmo.VenmoLineItem;
+import com.braintreepayments.api.venmo.VenmoLineItemKind;
+import com.braintreepayments.api.venmo.VenmoPaymentAuthRequest;
+import com.braintreepayments.api.venmo.VenmoPaymentAuthResult;
+import com.braintreepayments.api.venmo.VenmoPaymentMethodUsage;
+import com.braintreepayments.api.venmo.VenmoPendingRequest;
+import com.braintreepayments.api.venmo.VenmoRequest;
+import com.braintreepayments.api.venmo.VenmoResult;
 import com.google.android.material.button.MaterialButtonToggleGroup;
 
+import java.util.ArrayList;
+
 public class PaymentButtonsFragment extends BaseFragment {
 
     private PayPalButton payPalButton;
     private MaterialButtonToggleGroup payPalToggleGroup;
     private VenmoButton venmoButton;
+    private VenmoClient venmoClient;
+    private VenmoLauncher venmoLauncher;
```

Comment:
Are we able to internalize the VenmoClient and VenmoLauncher classes within the VenmoButton? This would reduce the number of integration steps a merchant would have to do to use the payment buttons.